burger2/script.js

```diff
@@ -0,0 +1,14 @@
+let img = document.querySelectorAll('img');
```

Comment:
it's a big mistake

now you added it for ALL images on the page, even on logo
so make your selectors more specific